### PR TITLE
[DO NOT MERGE] Full ORA staff grading changes for UX review

### PR DESCRIPTION
**Pull request by andy-armstrong**
@frrrances This PR contains all of the UX changes made for ORA staff grading, so please review in your copious remaining spare time. :-)

I've created a sandbox with a test ORA course here:

https://andy-armstrong.sandbox.edx.org/courses/course-v1:edx+ORA203+course/info

Note that the course has a new Staff Assessment section that you can use for full grading workflows. You can also apply re-grades to any assessment.

FYI @cahrens @explorerleslie @catong @efischer19 @dianakhuang @scottrish

### CODE REVIEW

**Comment by andy-armstrong** [comment]
@frrrances Note that GitHub says that the diff is too big, but fortunately all of the SASS changes are shown. Let me know if there still seems to be a problem.

**Comment by andy-armstrong** [comment]
@frrrances @cahrens My sandbox expired so I'm building a new one. Sorry!

**Comment by andy-armstrong** [comment]
@frrrances @cahrens The sandbox is all set now

**Comment by frrrances** [comment]
@andy-armstrong The UX looks good! I hit two small things: 

* I wasn't sure what this Classifier section was and the buttons are not styled correctly: 
screen shot 2015-12-22 at 1 49 47 pm

Unless authors/staff will know what this is, it might be helpful to have a little explanatory text to describe what the classifier section is and what the actions do.

* When I delete a student response I get a helpful message that I need to do another step to let them resubmit, but I'm not sure where or how to do that. More of a nice-to-have but could result in questions. Here is a screenshot: 

screen shot 2015-12-22 at 2 14 34 pm

Other than that the experience looks good, is easy to follow (or as easy as such a complicated problem can be), and acts as I would expect. I'll take a spin through the FED now.

File: openassessment/xblock/static/sass/oa/utilities/_developer.scss [hunk 1]
```diff
@@ -44,23 +44,14 @@ $link-hover: $edx-blue-l1 !default; // from our Pattern Library http://ux.edx.or
     padding: 10px;
 
     .ui-staff__button {
-      margin-left: ($baseline-v/2);
+      @extend %btn-reset;
+
```
Comment (frrrances) [inline openassessment/xblock/static/sass/oa/utilities/_developer.scss]: remove extra line

File: openassessment/xblock/static/sass/xb/elements/_controls.scss [hunk 2]
```diff
@@ -106,6 +106,16 @@
   border-radius: ($baseline-v/10);
 }
 
+%btn-reset {
```
Comment (frrrances) [inline openassessment/xblock/static/sass/xb/elements/_controls.scss]: nice extend!

File: openassessment/xblock/static/sass/oa/views/_oa-base.scss [hunk 3]
```diff
@@ -29,6 +29,56 @@
...
+    // STATE: actions has an error
+    &.has--error {
+
+      .message {
+        margin-bottom: $baseline-v;
+        @include text-align(left);
+      }
+    }
+
+    .action--submit {
```
Comment (frrrances) [inline openassessment/xblock/static/sass/oa/views/_oa-base.scss]: I think this is the style you want on the buttons that are looking funny but they don't have the list--actions parent.

**Comment by andy-armstrong** [comment]
@frrrances Thanks, I hadn't looked at example-based assessments. I have indeed messed up the styling on those buttons, so I'll fix those in a separate PR, along with any other feedback you have.

File: openassessment/xblock/static/sass/xb/elements/_controls.scss (hunk 2 above)
Comment (andy-armstrong) [inline openassessment/xblock/static/sass/xb/elements/_controls.scss]: I had a little help on that one. :-)

**Comment by frrrances** [comment]
FED looks good too. :+1:

**Comment by cahrens** [comment]
@andy-armstrong I noticed that when an ORA problem has a due date set, the rendering of the "tag" in the upper right corner is off. This may not be something you introduced, but I thought I would let you know just in case.

image

**Comment by cahrens** [comment]
@andy-armstrong the styling in the "Turbo Mode" (to continue assessing peers even after your required number is done) is broken.

image

**Comment by cahrens** [comment]
When the buttons have to appear on different lines because the browser is narrow, there is no space between them (true both on Chrome and Firefox):

image

**Comment by andy-armstrong** [comment]
@cahrens The issue you reported with the section labels is reproducible on the sandbox with ORA master. I've reported it as a bug:

https://openedx.atlassian.net/browse/TNL-3967

**Comment by andy-armstrong** [comment]
I'm addressing this feedback with https://github.com/edx/edx-ora2/pull/831, so I'm closing this PR.

**Pull request closed without merging**